build.assets/makefiles/base/agent/agent.mk

```diff
@@ -1,23 +1,7 @@
 .PHONY: all
 
-REPODIR := $(GOPATH)/src/github.com/hashicorp/serf
-OUT := $(ASSETDIR)/serf-$(SERF_VER)
-BINARIES := $(ROOTFS)/usr/bin/serf
+all:
```

Comment:
`$(BINARIES)` used to depend on `planet-agent.service` is that no longer needed?

Reply:
Good catch. This target should still depend on `planet-agent.service`.